keyboards/takashicompany/worm_eaten/readme.md

```diff
@@ -0,0 +1,44 @@
+# Worm Eaten
+
+![takashicompany/worm_eaten](https://i.imgur.com/8rIFpDR.jpg)
```

Comment:
This uses a smaller, but still high res version of the image, that is more appropriate for how github renders markdown.
```suggestion
![takashicompany/worm_eaten](https://i.imgur.com/8rIFpDRh.jpg)
```